webapp/services/get_beer.py

```diff
@@ -0,0 +1,37 @@
+import webapp2
+import json
+import logging
+from google.appengine.ext import ndb
+from google.appengine.api import search
+from models.beer import Beer
+
+class GetBeer(webapp2.RequestHandler):
+
+    def get(self):
+
+		beer_id = int( self.request.get('beerId', 0) )
+
+		beer = Beer.get_by_id(beer_id)
+
+		if beer != None:
+			beer_data = { 
+				"beerId" : beer.key.id(),
```

Comment:
Would be nice to have the same keys on both methods `get_all_beers` and `get_beer`. I prefer the format used in get_all_beers. Also I think is important to return the beer review and review_count also here since I think they'll be used on the app.

Reply:
You are right.. I did a lazy copy paste from the html methods. Should have checked twice.